### ISSUE TITLE
Unable to see who denied kudosu

### ISSUE THREAD

**Issue by Mirruuu** [ppy/osu-web#1864]
Should the "Denied from obtaining kudosu" statement include "by: insert nominator" to make it easier to discern who to talk to to get the kudosu back in the event you're unable to find another BN? Chances are the BN who you ask to reinstate it might ask the reason it was denied, and getting that reason from the person who denied it in the first place may lead to fewer complications. 

It's possible that one may lie about the reason their kudosu was denied to get it back and asking for confirmation from the BN that denied it would remove that possibility.

It would also open the possibility for the person whose kudosu was denied to dispute it with the denying BN. As of now, there's no way to know who to talk to. If a BN had their denial overwritten by another BN without any explanation over an issue they found valid, it wouldn't be the greatest feeling.

There's no mention of it in the log:
![image](https://user-images.githubusercontent.com/25852637/33444475-b4ad7a00-d5fa-11e7-86a0-3e1aa0a50d50.png)
And no mention of it in the post itself:
![image](https://user-images.githubusercontent.com/25852637/33444509-cb3ecc4c-d5fa-11e7-8df9-606d030cd6d6.png)

Tl;dr: Would be nice to see who denied kudosu.

**Comment by nanaya** [comment]
Maybe check the history tab of the discussion page.

**Comment by Mirruuu** [comment]
![image](https://user-images.githubusercontent.com/25852637/33445822-5e1dda1e-d5fe-11e7-9c99-fbb408353bb0.png)
There's still no mention of who did it.

**Comment by nanaya** [comment]
Apparently only visible to qat

**Comment by O1CDeif** [comment]
It's unfortunately not visible for QAT either. It'd be great to see who upvoted/downvoted or denied kudosu of any suggestion/problem in order to moderate the beatmap discussion properly and act whenever there's a case of allegedly abuse, like in this example:

![Image](https://puu.sh/yxurf/4b3e959d54.png)

The only thing we're able to see at the moment in terms of moderation are the deleted discussions and who did it.

**Comment by Naxesss** [comment]
That's not technically denying kudosu though, more related to downvotes, but it does have the same function and makes it a lot harder, if not impossible, for us to moderate this for sure. See the ["Downvoting removes kudosu and has no moderation options"](https://github.com/ppy/osu-web/issues/1803) issue for the above.

As for denying/allowing, it's visible as far as I can see:
![image](https://i.imgur.com/Be77yYm.png)
On the kudosu history you can't see who did it like you could with modding v1, but it's there I guess:
![image](https://i.imgur.com/9REy84V.png)
(also shown in the initial post before this issue was closed)

**Comment by peppy** [comment]
Downvotes and moderation tools will likely be added over the next week. It is a separate task to this one, though.
